portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -56,4 +61,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Redirect back to the HTML page.
         response.sendRedirect("/index.html");
     }
+
+    double getSentimentScore(String message) throws IOException {
+        Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+		LanguageServiceClient languageService = LanguageServiceClient.create();
```

Comment:
Can we initialize the client once and reuse it for multiple requests? In other words, the following should only be called once to process all the comments:

`LanguageServiceClient languageService = LanguageServiceClient.create();`